### PR TITLE
KCl fluorescence data analysis 

### PR DESCRIPTION
**Pull request by StephRZ**
https://github.com/sciunto/rayon/commit/9a60ac6447f58875d3fb5352600cf94cd85b094c

### CODE REVIEW

File: rayon/spectrum.py [hunk 1]
```diff
@@ -71,7 +71,7 @@ def get_peaks_data_1D(data_1D):
     Peaks are marked by their index (ie channel number)
 
     """
-    prominence = .05
+    prominence=0.05
```
Comment (sciunto) [inline rayon/spectrum.py]: Please, keep the spacing around signs :) (https://www.python.org/dev/peps/pep-0008/)

File: find_peaks.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python2
```
Comment (sciunto) [inline find_peaks.py]: It should be compatible with python3 as well. Please, remove the two first lines.

**Comment by sciunto** [comment]
In master, I added new files that will be ignored by git to avoid including png files in the future.

**Comment by StephRZ** [comment]
Ok, I'll try a new branch for the the latest changes. 
Sorry for the mistake. 

----- Mail original -----

De: "François Boulogne" <notifications@github.com> 
À: "sciunto/rayon" <rayon@noreply.github.com> 
Cc: "StephRZ" <stephan.rouziere@u-psud.fr>, "Author" <author@noreply.github.com> 
Envoyé: Mercredi 19 Septembre 2018 09:37:25 
Objet: Re: [sciunto/rayon] KCl fluorescence data analysis (#4) 



@sciunto requested changes on this pull request. 

Looks good (i'll need to run it, but I'm ready to merge). The only thing is that png files should not be included, it will blow up the size of the git repository. 

It's not so easy to remove these files once commited. My suggestion is to start a new branch, from scratch, and then add the python files. 

Feel free to ask if you need help. 

Many thanks! 


In rayon/spectrum.py : 
> @@ -71,7 +71,7 @@ def get_peaks_data_1D(data_1D):
     Peaks are marked by their index (ie channel number)

 

     """

-    prominence = .05

+    prominence=0.05 


Please, keep the spacing around signs :) ( https://www.python.org/dev/peps/pep-0008/ ) 


In find_peaks.py : 
> @@ -0,0 +1,82 @@
+#!/usr/bin/env python2 


It should be compatible with python3 as well. Please, remove the two first lines.

**Pull request closed without merging**